app/tests/test_serializer.py

```diff
@@ -857,6 +857,11 @@ def test_asset_serialization(self):
         logger.debug('python native:\n%s', pformat(python_native))
 
         # translate to JSON:
+        # remove read-only properties from python_native
+        read_only_fields = ["href", "created", "updated"]
+        for key in read_only_fields:
+            del python_native[key]
+
```

Comment:
This should be done later just before de-serialization in line 890 on the `python_native_back`